testing/firefox/fix-rust-target.patch

```diff
@@ -0,0 +1,11 @@
+--- a/build/moz.configure/rust.configure	2017-05-27 10:48:23.927528005 +0000
++++ b/build/moz.configure/rust.configure	2017-05-27 10:52:42.973790729 +0000
+@@ -130,7 +130,7 @@
+             ('arm', 'Linux'): 'armv7-unknown-linux-gnueabihf',
+             ('sparc64', 'Linux'): 'sparc64-unknown-linux-gnu',
+             ('x86', 'Linux'): 'i686-unknown-linux-gnu',
+-            ('x86_64', 'Linux'): 'x86_64-unknown-linux-gnu',
```

Comment:
This might prove problematic as there is no rust for any other architecture than x86_x64 at the moment, likely because the void linux supports only x86_64, see https://github.com/alpinelinux/aports/blob/master/community/rust/APKBUILD#L26.

We could use "--disable-rust" but that will stop working with version 54 (see https://bugzilla.mozilla.org/show_bug.cgi?id=1284816)